File: spryntax/website_static_build_tools/alg_creator/algs-c/pattern.c
#include <stdio.h>

void printPattern(int n) 
{
    int i, j;
    
    // Loop for each row
    for (i = 1; i <= n; i++) {
        // Print leading spaces
        for (j = 1; j <= n - i; j++) 
        {
            printf(" ");
        }
        
        // Print stars
        for (j = 1; j <= (2 * i - 1); j++) 
        {
            printf("*");
        }
        
        // Move to the next line
        printf("\n");
    }
}

int main() 
{
    int rows = 5;
    
    // Print the star pattern
    printPattern(rows);
    
    return 0;
}

File: spryntax/website_static_build_tools/alg_creator/algs-py/pattern.py
def print_star_pattern(n):
    # Print a pattern of stars in a right-aligned triangle form
    for i in range(1, n + 1):
        # Print leading spaces for right alignment
        print(" " * (n - i) + "*" * i)


# Driver Code for Star Pattern
rows = 5  # Number of rows in the pattern
print_star_pattern(rows)